src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private final Operation operation;
+    private final String fruitName;
+    private final int quantity;
+
+    public FruitTransaction(Operation operation, String fruitName, int quantity) {
+        this.operation = operation;
+        this.fruitName = fruitName;
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public String getFruitName() {
+        return fruitName;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+}
```

Comment:
The class fields are correctly encapsulated with private access modifiers. However, consider if you need to make them final. If you don't expect the properties of a transaction to change, it's good to keep them final. Otherwise, you might want to provide setters or consider a different design if mutability is required.